torch/fx/proxy.py

```diff
@@ -180,9 +180,9 @@ def create_proxy(self, kind: str, target: Target, args: Tuple[Any, ...], kwargs:
         if self.record_stack_traces and not proxy.node.stack_trace:
             user_frame = self._find_user_frame()
             if user_frame:
-                walk_stack_gen = traceback.walk_stack(user_frame)
-                summary = traceback.StackSummary.extract(walk_stack_gen)  # type: ignore[arg-type]
```

Comment:
interesting, I wonder why we did it this way originally

Reply:
Not sure... traceback.extract_stack(frame) would reverse the stack for user.